<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationRoom</class>
 <widget class="QWidget" name="CommunicationRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="recvBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>741</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>741</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>161</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>901</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>626</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
